neo4j/pom.xml

```diff
@@ -18,6 +18,9 @@
 
     <name>Hibernate OGM for Neo4j</name>
     <description>Persist objects in Neo4j</description>
+    <properties>
+        <jdkTargetForProcessor>1.8</jdkTargetForProcessor>
+    </properties>
```

Comment:
How come you need this here?

Reply:
By the way, we already have  a property called minJdkVersion set t java 1.8, so what is the purpose of this commit?